sample-java-client/README.md

```diff
@@ -0,0 +1,170 @@
+
+```shell
+mvn clean deploy
+```
+
+Refer to the [OSSRH Guide](http://central.sonatype.org/pages/ossrh-guide.html) for more information.
+
+### Maven users
+
+Add this dependency to your project's POM:
+
+```xml
+<dependency>
+  <groupId>io.swagger</groupId>
+  <artifactId>swagger-java-client</artifactId>
+  <version>1.0.0</version>
+  <scope>compile</scope>
+</dependency>
+```
+
+### Gradle users
+
+Add this dependency to your project's build file:
+
+```groovy
+compile "io.swagger:swagger-java-client:1.0.0"
+```
+
+### Others
+
+At first generate the JAR by executing:
+
+```shell
+mvn clean package
+```
+
+Then manually install the following JARs:
+
+* `target/swagger-java-client-1.0.0.jar`
+* `target/lib/*.jar`
+
+## Getting Started
+
+Please follow the [installation](#installation) instruction and execute the following Java code:
+
+```java
+
+import io.swagger.client.*;
+import io.swagger.client.auth.*;
+import io.swagger.client.model.*;
+import io.swagger.client.api.MoveTaskOrderApi;
+
+import java.io.File;
+import java.util.*;
+
+public class MoveTaskOrderApiExample {
+
+    public static void main(String[] args) {
+        
+        MoveTaskOrderApi apiInstance = new MoveTaskOrderApi();
+        Integer since = 56; // Integer | Only return move task orders updated since this time.
+        try {
+            MoveTaskOrders result = apiInstance.fetchMTOUpdates(since);
+            System.out.println(result);
+        } catch (ApiException e) {
+            System.err.println("Exception when calling MoveTaskOrderApi#fetchMTOUpdates");
+            e.printStackTrace();
+        }
+    }
+}
+
+```
+
+## Documentation for API Endpoints
+
+All URIs are relative to *http://localhost/prime/v1*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*MoveTaskOrderApi* | [**fetchMTOUpdates**](docs/MoveTaskOrderApi.md#fetchMTOUpdates) | **GET** /move-task-orders | Gets all move task orders where &#x60;isAvailableToPrime&#x60; is TRUE
+*MoveTaskOrderApi* | [**getMoveTaskOrderCustomer**](docs/MoveTaskOrderApi.md#getMoveTaskOrderCustomer) | **GET** /move-task-orders/{moveTaskOrderID}/customer | Gets the customer associated with a move task order ID
+*MoveTaskOrderApi* | [**updateMTOPostCounselingInformation**](docs/MoveTaskOrderApi.md#updateMTOPostCounselingInformation) | **PATCH** /move-task-orders/{moveTaskOrderID}/post-counseling-info | Updates move task order&#39;s post counseling information
+*MtoServiceItemApi* | [**createMTOServiceItem**](docs/MtoServiceItemApi.md#createMTOServiceItem) | **POST** /move-task-orders/{moveTaskOrderID}/mto-shipments/{mtoShipmentID}/mto-service-items | Creates MTO service items that is added to a Move Task Order and MTO Shipment
+*MtoShipmentApi* | [**updateMTOShipment**](docs/MtoShipmentApi.md#updateMTOShipment) | **PUT** /move-task-orders/{moveTaskOrderID}/mto-shipments/{mtoShipmentID} | Updates MTO shipment
```

Comment:
This endpoint is changing to `/mto-shipments/{:mtoShipmentID` per [MB-2476](https://dp3.atlassian.net/browse/MB-2476)